File: WebApp/Models/Karta.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class Karta
    {
        [Key]
        public int ID { get; set; }
        public int TipKarteId { get; set; }
        public TipKarte TipKarte { get; set; }
        public int TipPopustaId { get; set; }
        public TipPopusta TipPopusta { get; set; }
        public float Cena { get; set; }
        public bool Validna { get; set; }
        public DateTime DatumIzdavanja { get; set; }
    }
}

File: Frontend/DigitalPublicTransportSystem/src/app/models/karta.ts
export class Karta{
    Korisnik : string;
    TipKarte : string;
    TipPopusta : string;
    Cena : number;
    Datum : string;
    Id: string;

    constructor(korisnik?: string, tipKarte?: string, tipPopusta?: string, cena?: number, datum?: string, id?: string)
    {
        this.Korisnik = korisnik;
        this.TipKarte = tipKarte;
        this.TipPopusta = tipPopusta;
        this.Cena = cena;
        this.Datum = datum;
        this.Id = id;
    }
}